foss4g2023.html

```diff
@@ -790,6 +790,16 @@ <h3>Turning git commit entries into a release page</h3>
     <p>by Vaclav Petras (procedure support by Markus Neteler)</p>
 </section>
 
+<section>
+    <h2>New docker hub organization</h2>
+    <br>
+    <h3>mundialis -> OSGeo</h3>
+    <ul>
+        <li>The GRASS GIS docker images are currently being moved from "mundialis" to the <a href="https://hub.docker.com/u/osgeo">OSGeo Docker Hub organization</a></li>
```

Comment:
<li>The GRASS GIS docker images are currently being moved from <a href="https://hub.docker.com/u/mundialis">mundialis</a> to the <a href="https://hub.docker.com/u/osgeo">OSGeo Docker Hub organization</a></li>
```suggestion
        <li>The GRASS GIS docker images are currently being moved from <a href="https://hub.docker.com/u/mundialis">mundialis</a> to the <a href="https://hub.docker.com/u/osgeo">OSGeo Docker Hub organization</a></li>
```